Rebus/Pipeline/Receive/DispatchIncomingMessageStep.cs

```diff
@@ -41,18 +41,17 @@ public DispatchIncomingMessageStep(IRebusLoggerFactory rebusLoggerFactory)
         /// </summary>
         public async Task Process(IncomingStepContext context, Func<Task> next)
         {
-            var invokers = context.Load<HandlerInvokers>();
             var handlersInvoked = 0;
+            var invokers = context.Load<HandlerInvokers>();
             var message = invokers.Message;
 
-            var messageId = message.GetMessageId();
-            var messageType = message.GetMessageType();
+            var messageLabel = message.GetMessageLabel();
 
             // if dispatch has already been aborted (e.g. in a transport message filter or something else that
             // was run before us....) bail out here:
             if (context.Load<bool>(AbortDispatchContextKey))
             {
-                _log.Debug("Skipping dispatch of message {messageType} {messageId}", messageType, messageId);
```

Comment:
With a structured logging library like [Serilog](https://github.com/serilog/serilog), this will log an event accompanied by the following data:
```
{
	"messageType": "SomeType, SomeAssembly",
	"messageId": "4393umessageID"
}
```
making it possible to filter and aggregate logs using these values further down the line (e.g. in Kibana, if you're using the ELK stack).